network.go

```diff
@@ -150,3 +165,89 @@ func (c *Client) DeleteNetwork(id string) (*SimpleResponse, error) {
 
 	return c.DecodeSimpleResponse(resp)
 }
+
+// GetSubnet gets a subnet with ID
+func (c *Client) GetSubnet(networkID, subnetID string) (*Subnet, error) {
+	resp, err := c.SendGetRequest(fmt.Sprintf("/v2/networks/%s/subnets/", networkID) + subnetID)
```

Comment:
you can also do: `fmt.Sprintf("/v2/networks/%s/subnets/%s", networkID, subnetID)`